Bio/PDB/PDBList.py

```diff
@@ -293,8 +293,7 @@ def retrieve_pdb_file(self, pdb_code, obsolete=False, pdir=None, file_format=Non
         return final_file
 
     def update_pdb(self, file_format=None):
-        """
-        I guess this is the 'most wanted' function from this module.
+        """I guess this is the 'most wanted' function from this module.
```

Comment:
That's not suitable as a one line summary - it could stay in the main body of the docstring.

How about ``Update your local copy of the PDB files.`` here?